workspaces/announcements/package.json

```diff
@@ -34,9 +37,9 @@
     "directory": "workspaces/announcements"
   },
   "devDependencies": {
-    "@backstage/cli": "^0.29.0",
+    "@backstage/cli": "^0.29.4",
     "@backstage/e2e-test-utils": "^0.1.1",
-    "@backstage/repo-tools": "^0.10.0",
+    "@backstage/repo-tools": "^0.12.0",
     "@changesets/cli": "^2.27.1",
     "@spotify/prettier-config": "^12.0.0",
```

Comment:
This should be removed

Reply:
Knip should be under `devDependencies`

Reply:
and `"prettier": "@spotify/prettier-config"` should be `"prettier": "@backstage/cli/config/prettier"`